CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Change Log
 
+## Version 1.36.1
+* Release date: April 22, 2022
+* Release status: General Availability
+## What's new in this version
+- April Hotfix addressing these issues https://github.com/microsoft/azuredatastudio/milestone/88?closed=1.
+
+| Platform																|
```

Comment:
@erpett is the plan to include links to each build now?  or only the current one?  I wasn't sure what you had in mind with the link section in this document.

Reply:
The instructions I got when i took over release management is that we only include the latest links in the changelog and the fwlinks for the hotfix should be the same as yesterday's release

Reply:
So I have just been moving the "old" notes to below the links and adding the new notes above it.

Reply:
thanks @erpett.  I'll update the PR to remove the links from 1.36.0 entry in that case.